samples/sampler/stories/default/Button.js

```diff
@@ -33,22 +35,39 @@ const threeWayBoolean = (value) => {
 	}
 };
 
-const SkinnedButton = Skinnable(
-	{prop: 'skin'},
-	({skin, ...rest}) => {
+const SkinnedButtonBase = kind({
+	name: 'SkinnedButton',
+
+	propTypes: {
+		skin: PropTypes.string
+	},
+
+	render: ({skin, ...rest}) => {
 		let icons = skin === 'silicon' ? ['', ...iconListSilicon] :  ['', ...iconList];
 
 		return (
 			<Button
 				{...rest}
+				icon={select('icon', icons, Config)}
+			/>
+		);
+	}
+});
+
+const SkinnedButton = Skinnable({prop: 'skin'}, SkinnedButtonBase);
+
+storiesOf('Agate', module)
+	.add(
+		'Button',
+		() => (
+			<SkinnedButton
 				animateOnRender={boolean('animateOnRender', Config)}
```

Comment:
I'm not sure if it's better to be in the same position as icon. Please judge when merge. @seunghoh 

Also, if I click "ShowInfo button" on the right side, it display as Skinnable. This seems unavoidable, but I would like to check if there is a solution in other JIra issue. Could you make it? @adrian-cocoara-lgp

Reply:
By adding `SkinnedButton.displayName = 'Button'`, clicking on "show info" button would display <Button> instead of <Skinnable>. What do you think of this approach?

Reply:
OK. Looks better to me.